### PR TITLE
[bookmarklet] Insert Instagram thumbnail automatically

### PR DESCRIPTION
**Pull request by clbn**

### CODE REVIEW

File: public/js/app/views/BookmarkletView.js [hunk 1]
```diff
@@ -18,6 +18,14 @@ define(["app/app",
       })
     },
 
+    selectInstagramThumbnail: function() {
+      var re = /https?:\/\/www\.instagram\.com\/p\/([\w-]{11})\//i;
```
Comment (davidmz) [inline public/js/app/views/BookmarkletView.js]: 11 is a wrong number, Instagram ID's doesn't have a fixed length. For example, first image in Instagram: https://www.instagram.com/p/C/. So I suggest to use '+', not '{11}'.

**Comment by davidmz** [comment]
We need to test it, this link isn't a direct link to image file but a redirect link. Can node follow redirects?

**Comment by clbn** [comment]
@davidmz Sure, I tested it, a redirect works on my instance just fine.

I'll fix the ID length, thanks.

**Comment by clbn** [comment]
@davidmz PR has been updated.

**Pull request merged**